src/main/java/com/example/CarDealershipSpring/models/UserInterface.java

```diff
@@ -0,0 +1,34 @@
+package com.example.CarDealershipSpring.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserInterface {
+    Dealership dealership;
+
+    public void display() {
```

Comment:
So if we're going to just directly use the repo in the console app then there is kind of not really a need for this class at all.  However, if you are going to have multiple Dealerships each with their own inventory, then you would need to plug the repo in here so that you can query using the Dealership id along with the other information

Reply:
plug it in like this?

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserInterface {
    Vehicle Repository = new VehicleRepository();
    Dealership dealership;

Reply:
You can autowire the vehicle repository